traitsui/qt4/list_editor.py

```diff
@@ -742,21 +750,3 @@ def _menu_action(self, event, name=""):
         else:
             # TODO: Fix tab order based on the context_object's list
             self.control.addTab(self._pagewidgets[name], name)
-
-    def _context_menu_requested(self, event):
```

Comment:
Why is this removed?  This seems unrelated to the main change?

Reply:
This is duplicate code from lines [728-744](https://github.com/enthought/traitsui/blob/35f0510ca3090574f121786221548b09ac1882a8/traitsui/qt4/list_editor.py#L728-L744) above.

Would you like me to address this in a separate PR instead?

Reply:
Ah.  That makes sense - it wasn't obvious from the diff.  It's fine to do drive-by fixes like this, just mention it in the PR description at the top that you are also doing this change.